com.bitwig.BitwigStudio.yaml

```diff
@@ -25,56 +25,43 @@ finish-args:
   # For desktop notifications
   - --talk-name=org.freedesktop.Notifications
 
-  # Bitwig Studio specific folders
-  - --filesystem=~/.BitwigStudio
-  - --filesystem=~/Bitwig Studio
-
-  # Standard Plugin folders
-  - --filesystem=~/.vst
-  - --filesystem=~/.vst3
-  - --filesystem=/usr/lib64/vst
```

Comment:
Why don't we need this filesystem?

Reply:
simple because there is absolutely no guarantee this works. It's a bad idea.

Reply:
What if it could work? I have no idea how u-he would distribute their plugins using flatpak.